### DISCUSSION TITLE
How to change git auth info?

### DISCUSSION THREAD

**Discussion by radesavkovic** [community/community#61084] [Discussions] [answered]
### Select Topic Area

Question

### Body

I'm trying to commit and push for a invited repo but can't do it.
It says not permitted for the action.
It seems to git auth is different what I invited auth info.
How to fix it?

**Comment by Blockchain1207** [comment, marked as answer]
I experienced with the issue.
It's easy. Just follow the steps.
You should install git cli.
And run cmd as follows.

#gh auth login

And follow the questions and answer.
You can login with website or token.

That's all.

**Comment by radesavkovic** [comment]
It's quite easy.
Thank you.
